ui/.neutrinorc.js

```diff
@@ -160,6 +160,8 @@ module.exports = {
     (neutrino) => {
       neutrino.config.resolve
         .alias.set('taskcluster-ui', resolve(__dirname, 'src/'));
+      neutrino.config.resolve
+        .alias.set('react-dom', '@hot-loader/react-dom');
```

Comment:
I think we'll need to import this library and remove the old one:

```jsx
yarn remove react-dom
yarn add @hot-loader/react-dom
```

Reply:
Tried this but dosnt't seem to work. Still getting the same errors like below -  
![image](https://user-images.githubusercontent.com/22626144/59014386-1f97ff80-885a-11e9-80f6-90719e8bd6e4.png)

Also tried [this](https://github.com/hot-loader/react-dom#yarn-any-other-system) but cant see any change in`package.json` after i add packages like after `yarn add @hot-loader/react-dom` or `yarn add react-error-boundary`. This is the log of the `yarn add *` command: 
![image](https://user-images.githubusercontent.com/22626144/59014493-64bc3180-885a-11e9-9790-62183fb8db34.png)

Can you make something of the issue from the logs?

Reply:
It looks like the issue is coming from the python version. Try giving [this](https://github.com/taskcluster/taskcluster-web/issues/306#issuecomment-446611865) a try.

Reply:
Solved these issues by updating the global `windows-build-tools` module with `yarn global add windows-build-tools`. Now also able to install modules like `react-error-boundary`